TSG.md

```diff
@@ -0,0 +1,207 @@
+# Common Mount Problems
+**Logging**
+
+Please ensure logging is turned on DEBUG mode when trying to reproduce an issue.
+This can help in many instances understand what the underlying issue is.
+
+A useful setting in your configuration file to utilize when debugging is `sdk-trace: true` under the azstorager component. This will log all outgoing REST calls.
+
+**1. Error: fusermount: failed to open /etc/fuse.conf: Permission denied**
+
+Only the users that are part of the group fuse, and the root user can run fusermount command. In order to mitigate this add your user to the fuse group.
+
+```sudo addgroup <user> fuse```
+
+**2. failed to mount : failed to authenticate credentials for azstorage**
```

Comment:
In case of private AAD endpoint (private MSI endpoitns) user need to configure them in environment variable. In v1 it used to work with config file setting but with v2 it will not and env variable is must now.